README.md

```diff
@@ -1 +1,63 @@
-# module-ballerina-system
\ No newline at end of file
+Ballerina System Library
+===================
+
+  [![Build](https://github.com/ballerina-platform/module-ballerina-system/workflows/Build%20master%20branch/badge.svg)](https://github.com/ballerina-platform/module-ballerina-system/actions?query=workflow%3ABuild)
```

Comment:
Shall we rename this GitHub action as just build? Else the badge will be really long

https://github.com/ballerina-platform/module-ballerina-system/blob/stdlib-configs/.github/workflows/build-master.yml#L1

Reply:
Renamed as suggested